src/plugins/saved_objects_management/public/plugin.ts

```diff
@@ -105,6 +105,9 @@ export class SavedObjectsManagementPlugin
         defaultMessage: 'Saved Objects',
       }),
       order: 1,
+      meta: {
```

Comment:
Added for testing purposes. The intent is to let plugin developers add their own keywords.